README.md

```
@@ -36,49 +43,73 @@ An AI-powered email drafting assistant that generates **polite, relationship-awa
+
+> **Branch**: `iteration-1-demo`
 
-Sync Gradle
+```bash
+git switch iteration-1-demo
+cp .env.example .env
+docker compose up -d --build
+```
 
-Select an emulator/device (e.g., Pixel 7, API 34)
+Default URL: http://localhost/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)